### PR TITLE
feat (apps/pano): use dialog for post deletion

### LINKED ISSUE #569: feat(pano): Add dialog for post deletion confirmation

### PR DESCRIPTION
**Pull request by Murat-tunca**
Adds the modal required for post deletion using the dialog component

closes #569 
image

### CODE REVIEW

**Comment by Ketcap** [comment]
there are still some things needs to be done so I'm converting it to draft.

1. Backend is missing
2. Variable namings are not fixed
3. Custom "username : cancan" is still there 🤣
4. **rebase** branch from develop please

**Comment by cansirin** [comment]
> there are still some things needs to be done so I'm converting it to draft.
> 
> 1. Backend is missing
> 2. Variable namings are not fixed
> 3. Custom "username : cancan" is still there 🤣
> 4. **rebase** branch from develop please

Thanks so much for converting it to draft! This is kind of a placeholder implementation to make layout ready for the logic so we'll be integrating backend when the development is finished for those operations 👍🏻

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx [hunk 1]
```diff
@@ -66,7 +87,24 @@ export const MoreOptionsDropdown = ({ post, shareUrl }: Props) => {
...
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Silmek istediğine emin misin?</DialogTitle>
+          <DialogDescription>
+            Eğer bu gönderiyi silersen, bu işlemi geri alamazsın.
+          </DialogDescription>
+        </DialogHeader>
+        <DialogFooter>
+          <Button variant="outline" type="submit">
+            Hayir
```
Comment (cansirin) [inline apps/kampus/app/pano/features/post-list/MoreOptions.tsx]:
```suggestion
            Hayır
```

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx [hunk 2]
```diff
@@ -25,39 +34,51 @@ type Post = {
...
+};
+
+function canUserEdit(user: User, post: Post) {
+  console.log(user, post);
+  return true;
+}
 
 export const MoreOptionsDropdown = ({ post, shareUrl }: Props) => {
-  // const user = useUserContext();
+  const user = { username: "cancan" };
```
Comment (cansirin) [inline apps/kampus/app/pano/features/post-list/MoreOptions.tsx]:
```suggestion
  const user = { username: "John Doe" };
```

**Review by cansirin — APPROVED** [review]
💯

**Pull request merged**